deployment/hasura/metadata/databases/tables/actions/action_run.yaml

```
@@ -0,0 +1,47 @@
+table:
+  name: action_run
+  schema: actions
+configuration:
+  custom_name: "action_run"
+object_relationships:
+  - name: action_definition
+    using:
+      foreign_key_constraint_on: action_definition_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
```

`aerie_admin` should also have insert column restrictions (see comment on `action_definition` about not letting them set, for example, `id`). I recommend the following:

```suggestion
      columns: [settings, parameters, action_definition_id]
```